start-site/src/main/resources/application.yml

```diff
@@ -199,12 +207,26 @@ initializr:
       spring-releases:
         name: Spring Releases
         url: https://repo.spring.io/release
+      spring-milestones:
+        name: Spring Milestones
+        url: https://repo.spring.io/milestone
       sonatype-snapshots:
         name: Sonatype Snapshots
         url: https://oss.sonatype.org/content/repositories/snapshots/
         releasesEnabled: false
         snapshotsEnabled: true
   dependencies:
+    - name: Architecture
```

Comment:
I'd not create a new group for a single entry for now. Can you please move it to the Developer Tools group?